meta-lmp-base/classes/fip-utils.bbclass

```diff
@@ -246,12 +190,10 @@ do_deploy:append:class-target() {
                 # Generate certificates
                 ${CERTTOOL} -n --tfw-nvctr 0 --ntfw-nvctr 0 --key-alg ecdsa --hash-alg sha256 \
                         --rot-key ${sign_key} \
-                        --rot-key-pwd ${FIP_SIGN_KEY_PASS} \
```

Comment:
That is already possible by having the same key but provided by the factory (via another repo or via the factory specific lmp-manifest). Since both the key and the password would have to be given/provided to the build, I don't really see how this could really protect the user (I think it was added just because cube creates a key with password by default, but we don't need that here).

Later on our idea is to support cloudhsm instead, which would be used by u-boot/kernel/optee and atf.

Reply:
got it, thanks!